### DISCUSSION TITLE
Can we contribute by adding more monsters to the bestiary?

### DISCUSSION THREAD

**Discussion by Catacomba** [Obsidian-TTRPG-Community/fantasy-statblocks#69] [Q&A] [answered]
Hi, I have a question in regards to expanding the bestiary to include other books, for example Volos guide to monsters. 
Is there any specific reason (aside from the amount of work it takes to add these) that these are not included?
Could contributors be allowed to add these? 

Thank you in advance for the awnser, and big thanks for this plugin, its amazing :)

**Comment by valentine195** [comment, marked as answer]
You can only legally distribute the SRD.
